src/tutorial/testing.md

```diff
@@ -1,9 +1,9 @@
 # Testing
 
 Over the decades of people doing software development
-one truth has been found:
+a pattern has emerged:
 Untested software rarely works.
-(Many people would go so far and add
+(Many people would go that far and add
```

Comment:
"so" is correct imho

Reply:
it is correct but the sentence looks odd

Reply:
You're right, I only looked at the diff. How about:

> Many people would go as far as saying: "Most tested software doesn't work either."